blitzloop/play.py

```diff
@@ -91,15 +96,17 @@ def render():
 pause = False
 
 def key(k):
+    if type(k) is bytes:
+        k = int.from_bytes(k, sys.byteorder)
```

Comment:
Or just k = k[0] for the same effect.

But why not just prefix all the strings below with b, as in b'\033'? That's the straightforward py2→py3 way.